strawberry/cli/commands/export_schema.py

```diff
@@ -1,14 +1,19 @@
-import importlib
-
 import click
 
+from strawberry import Schema
 from strawberry.printer import print_schema
+from strawberry.utils.importer import import_module_symbol
 
 
 @click.command(short_help="Exports the schema")
-@click.argument("selector", type=str)
-def export_schema(selector: str):
-    module_name, symbol_name = selector.rsplit(":", 1)
-    schema_module = importlib.import_module(module_name)
-    schema_symbol = getattr(schema_module, symbol_name)
+@click.argument("schema", type=str)
+def export_schema(schema: str):
+    try:
+        schema_symbol = import_module_symbol(schema, default_symbol_name="schema")
+    except (ImportError, AttributeError) as exc:
+        message = str(exc)
+        raise click.BadArgumentUsage(message)
```

Comment:
I actually find the ImportError and AttributeError messages quite helpful. They look like this:

`Error: No module named 'fake_module'`

`Error: module 'sample_package.sample_module' has no attribute 'schema'`

Reply:
You'll still get the raw `ImportError` and `AttributeError` exceptions because of the `from exc`. I'm not super committed to my suggestion -- I'll leave the decision up to you.

Reply:
I tested both versions and still prefer mine. It includes the error message but does not overwhelm the user with the stacktrace. (Also it turned out that click swallows the ImportError/AttributeError message when using `from exc`).